### PR TITLE
updated JEC to Summer15_50nsV4

### PR DESCRIPTION
**Pull request by jshlee**
revert backed to MET inheriting from cat::Particle 
Summer15_50nsV4

updated jet ID
    bool looseJetID_;
    bool tightJetID_;
    bool tightLepVetoJetID_;

added JEC Uncertainty to catJets

### CODE REVIEW

File: CatAnalyzer/test/create-batch [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+            print "@@ Jobs are prepared. You can submit jobs with following command:"
+            print "cd %s;./submit.sh" % (self.jobDir)
+
+## Create job configure
+jobConfig = TheJobConfig(opts)
+jobConfig.initialiseWorkspace()
+jobConfig.archive()
+jobConfig.submit()
+
+print "@@ Done."
```
Comment (jhgoh) [inline CatAnalyzer/test/create-batch]: What is different to the central create-batch script?
I'd like to merge this if there's any improvement.

File: CatAnalyzer/test/run_TtbarDiLeptonAnalyzer.py [hunk 2]
```diff
@@ -5,28 +5,47 @@
...
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(-1) )
+
+###### discard this line when use for data sample ##########
+process.partonTop = cms.EDProducer("PartonTopProducer",
+    genParticles = cms.InputTag("prunedGenParticles"),
+    jetMinPt = cms.double(20),
+    jetMaxEta = cms.double(2.5),
+    jetConeSize = cms.double(0.4),
+)
+###### discard this line when use for data sample ##########
```
Comment (jhgoh) [inline CatAnalyzer/test/run_TtbarDiLeptonAnalyzer.py:17]: Why don't you use

```
process.load("CATTools.CatProducers.pseudoTop_cff")
```

File: CatAnalyzer/test/run_TtbarDiLeptonAnalyzer.py [hunk 3]
```diff
@@ -5,28 +5,47 @@
...
-#process.source.fileNames.append('file:/store/group/CAT/SingleMuon/v7-3-0_Run2015B-PromptReco-v1/150720_060727/0000/catTuple_1.root')
-process.source.fileNames.append('file:/store/group/CAT/WZ_TuneCUETP8M1_13TeV-pythia8/v7-3-0_RunIISpring15DR74-Asympt50ns_MCRUN2_74_V9A-v2/150720_070007/0000/catTuple_1.root')
+"""
+for i in xrange(1,101):
+#    process.source.fileNames.append('file:/cms/data/xrd/store/user/jlee/TTJets_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/cat74v2_RunIISpring15DR74-Asympt50ns_MCRUN2_74_V9A-v1/150713_164609/0000/catTuple_%d.root' % i)
+	process.source.fileNames.append('file:/cms/scratch/CAT/TTJets_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/v7-3-0_RunIISpring15DR74-Asympt50ns_MCRUN2_74_V9A-v1/150720_065809/0000/catTuple_%d.root'%i)
+"""
+#process.source.fileNames.append('file:/cms/scratch/CAT/MuonEG/v7-3-0_Run2015B-PromptReco-v1/150720_060935/0000/catTuple_1.root')
+#process.source.fileNames.append('file:/cms/scratch/CAT/WW_TuneCUETP8M1_13TeV-pythia8/v7-3-2_RunIISpring15DR74-Asympt50ns_MCRUN2_74_V9A-v1/150805_203816/0000/catTuple_1.root')
```
Comment (jhgoh) [inline CatAnalyzer/test/run_TtbarDiLeptonAnalyzer.py:22]: You can simply remove these lines :-)

File: CatAnalyzer/plugins/memo [hunk 4]
```diff
@@ -0,0 +1,8 @@
+22/07/15
+
+''For TtbarDilepton Analysis''
+-> have to do something about 3 lepton situation. (veto? or choose by pT?)
+-> change the name of jobs, so can draw stacks easily.
+-> try to draw 'njet', 'nbjet' plot, and compare with Dr. roh's or jason's results.
+-> would be much nicer if set channel numbers equal for CAT and GEN, PartonTOP like ee 1, em 2, mm 3 (for mode too)
+
```
Comment (jhgoh) [inline CatAnalyzer/plugins/memo]: Do we need this file?

File: CatAnalyzer/test/run_TtbarDiLeptonAnalyzer.py (hunk 2 above)
Comment (jshlee) [inline CatAnalyzer/test/run_TtbarDiLeptonAnalyzer.py:17]: because pseudoTop_cff also makes pseudoTop

**Comment by jshlee** [comment]
i've cleaned up
and will use these updates for 735 prod soon

**Comment by jshlee** [comment]
will merge since this will be v7-3-5

**Pull request merged**